articles/how-to-python-qdk-local.md

```diff
@@ -77,7 +77,11 @@ Once imported, the `qsharp` package allows Q# namespaces to appear as Python mod
     ```output
     Hello, quantum world!
     ```
-    
+
+> [!NOTE]
+> If you hit an error `Unable to load shared library 'Microsoft.Quantum.Simulator.Runtime' or one of its dependencies`, then you need to install the `libomp` library, as the OpenMP support library is required by the quantum simulator of the QDK. On Ubuntu, you can do so by running `sudo apt install libomp5-11`, or `yum install libomp` on CentOS. For other distributions, please refer to your particular package manager.  
```

Comment:
'then you need to install' - I'd think that this is only one of the possible reasons for getting 'Unable to load'. So I suggest something like "then you may be missing libomp, as the OpenMP..."

Reply:
Thanks, changed.